docs/source/install.rst

```diff
@@ -4,6 +4,89 @@
 Installing Volttron
 ===================
 
+Installing Linux Virtual Machine
+--------------------------------
+Volttron requires a Linux system to run. For most this will require virtual machine (VM). This section will describe the
```

Comment:
For windows users this will require a virtual machine (VM). 

VOLTTRON not Volttron in most places.